charts/scalardb-cluster/values.yaml

```
@@ -317,3 +317,17 @@ scalardbCluster:
         - localhost
       # -- Issuer references of cert-manager.
       issuerRef: {}
+
+  encryption:
+    # -- Enable encryption at rest. You must set this to `true` if you're using the encryption feature in ScalarDB Cluster.
+    enabled: false
+    # -- Type of encryption. You must set this value to the same value as "scalar.db.cluster.encryption.type" for ScalarDB Cluster.
+    type: ""
+    # -- Vault-specific configurations.
+    vault:
+      # -- TLS configurations to provide access from ScalarDB Cluster to Vault by using TLS. If you're using HashiCorp Cloud Platform (HCP) Vault Dedicated, you don't need to set these TLS configurations even though HCP Vault Dedicated uses HTTPS. You need to set these TLS configurations only if you're using HashiCorp Vault deployments other than HCP Vault Dedicated, like a self-hosted HashiCorp Vault.
```

```suggestion
      # -- TLS configurations to provide access from ScalarDB Cluster to Vault by using TLS. If you're using HashiCorp Cloud Platform (HCP) Vault Dedicated, you don't need to set these TLS configurations because HCP Vault Dedicated uses HTTPS. You need to set these TLS configurations only if you're using HashiCorp Vault deployments other than HCP Vault Dedicated, like a self-hosted HashiCorp Vault.
```
`even though` part sounds pretty weird to me.

---

@feeblefakie 
Thank you for your review and discussion in the meeting!
Based on the discussion result, I updated this description in 27b81d9d4242d551b500a7dcc31e999fa1114111.
Please take a look again when you have time! 🙇‍♂️

@josh-wong 
As I described above, I updated this description.
Could you please take a look again when you have time? 🙇‍♂️